src/goBuild.ts

```diff
@@ -23,13 +23,18 @@ export function buildCode(buildWorkspace?: boolean) {
 
 	let documentUri = editor ? editor.document.uri : null;
 	let goConfig = vscode.workspace.getConfiguration('go', documentUri);
+	statusBarItem.show();
+	statusBarItem.text = 'Building in progress';
 	outputChannel.clear();
-	outputChannel.show();
 	outputChannel.appendLine('Building in progress...');
 	goBuild(documentUri, goConfig, buildWorkspace)
-		.then(errors => handleDiagnosticErrors(editor ? editor.document : null, errors, vscode.DiagnosticSeverity.Error))
+		.then(errors => {
+				handleDiagnosticErrors(editor ? editor.document : null, errors, vscode.DiagnosticSeverity.Error);
+				statusBarItem.text = 'Build Complete';

```

Comment:
I wonder if we should not show anything when the build/lint/vet process via commands is complete.
We might end up in a case where the status bar says "complete" but the on save build/lint/vet processes might be running.